hbase-common/src/test/java/org/apache/hadoop/hbase/io/util/TestLRUDictionary.java

```diff
@@ -81,9 +83,8 @@ public void testPassingSameArrayToAddEntry() {
 
   @Test
   public void testBasic() {
-    Random rand = new Random();
     byte[] testBytes = new byte[10];
-    rand.nextBytes(testBytes);
+    RNG.nextBytes(testBytes);
```

Comment:
What is the real gain here? We just the the random once, and I suggest that we just use ThreadLocalRandom instead...

Reply:
Sure, let's use ThreadLocalRandom.current() instead of the static Random instance.